aion/web/controllers/question_transactions.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Aion.QuestionTransactions do
+  @moduledoc """
+  Implements transactions used by Aion.QuestionController
+  """
+  use Aion.Web, :controller
+
+  alias Aion.Question
```

Comment:
Also, these aliases could be rewritten to be a one-line job, like this:
```
alias Aion.{Question, Answer, ...}
```
I usually chose this option as it's only an import statement, it's still pretty clear and this way it takes less space. Let me know, maybe there's a specific reason to opt for multiple-line imports like here, I'm curious.